lib/puppet/face/facts.rb

```diff
@@ -1,5 +1,23 @@
 require 'puppet/indirector/face'
 require 'puppet/node/facts'
+require 'puppet/util/fact_dif'
```

Comment:
nit: Since the action is also `diff` with double f, I would name the util file the same way.